Code/autopkglib/github/__init__.py

```diff
@@ -163,8 +163,8 @@ def search_for_name(
         results_limit: int = 100,
     ):
         """Search GitHub for results for a given name."""
-
         query = f"q={name}+extension:recipe+user:{user}"
+        query = query.replace(" ", "%20")
```

Comment:
Instead of doing a straight string replace, try using `urllib.parse.quote()` instead:
https://docs.python.org/3/library/urllib.parse.html#urllib.parse.quote

Reply:
https://github.com/autopkg/autopkg/pull/664/commits/74b4665bbb5e928d6364f0f7c9634ae2c4190523